lib/TerminalDisplay.cpp

```diff
@@ -431,12 +440,14 @@ TerminalDisplay::TerminalDisplay(QWidget *parent)
   setLayout( _gridLayout );
 
   new AutoScrollHandler(this);
+  autoHideMouseAfter(1000);
 }
 
 TerminalDisplay::~TerminalDisplay()
 {
   disconnect(_blinkTimer);
   disconnect(_blinkCursorTimer);
+  disconnect(gs_hideMouseTimer);
```

Comment:
Missing nullity check.

Reply:
Fwwi, delete(static_cast<QObject*>(nullptr)) is fine, but we can add a redundant check.
Wrt the pointer shenanigans below, we could also generate the pointer unconditionally (except when not leaking it is preferred in which case we'll have to share it)

Reply:
I just suggested it for the code reader ("This timer is different from the others."). Otherwise, it wasn't important.